index.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 let headerMaxLength = 72
+let prLength = 4
```

Comment:
The variable name isn't that telling... The number `4` is the number of chars added besides the PR ID length, so maybe something like `extraCommitMsgChars`?